docs/advanced.rst

```diff
@@ -422,49 +656,24 @@ A smaller heartbeat period increases the number of overhead messages in the netw
 but speeds up the system response when a piece of data is lost.
 
 Non-strict reliability
-""""""""""""""""""""""
+----------------------
 
 Using a strict reliability, configuring :ref:`history-qos` kind as ``KEEP_ALL``, determinates all samples have to be
 received by all subscribers.
 This implicates a performance decrease in case a lot of samples are dropped.
 If you don't need this strictness, use a non-strict reliability, i.e. configure :ref:`history-qos` kind as ``KEEP_LAST``.
 
 Slow down sample rate
-^^^^^^^^^^^^^^^^^^^^^
+=====================
 
 Sometimes publishers could send data in a too high rate for subscribers.
 This can end dropping samples.
 To avoid this you can slow down the rate using :ref:`flow-controllers`.
 
 .. _whitelist-interfaces:
```

Comment:
I think this should also be moved upwards

Reply:
Done